exporters/otlp/common/src/test/java/io/opentelemetry/exporter/internal/retry/RetryInterceptorTest.java

```diff
@@ -118,6 +127,22 @@ void interrupted() throws Exception {
     }
   }
 
+  @Test
+  void connectTimeout() throws Exception {
+    when(random.get(anyLong())).thenReturn(1L);
+    doNothing().when(sleeper).sleep(anyLong());
+
+    // Connecting to a non-routable IP address to trigger connection timeout
+    assertThatThrownBy(
+            () ->
+                client.newCall(new Request.Builder().url("http://10.255.255.1").build()).execute())
+        .isInstanceOf(SocketTimeoutException.class)
+        .hasMessage("Connect timed out");
```

Comment:
test failure seems to suggest this should be a lower-cased "connect" in this message.

Reply:
Weird.. the message case seems to vary by JVM version 🤔

Reply:
oh, that's extra fun!  :)